insta-unfollower.py

```diff
@@ -179,7 +190,7 @@ def get_following_list():
 
 def unfollow(user):
     response = session.get(profile_route % (instagram_url, user['username']))
-    time.sleep(random.randint(2, 4))
+    time.sleep(random.randint(2, 8))
```

Comment:
Don't change this for now, better to make another PR/Commit reviewing all timers